UnityProject/Assets/Scripts/Player/Mind.cs

```diff
@@ -48,7 +48,7 @@ public class Mind : NetworkBehaviour, IActionGUI
 	public PlayerInfo ControlledBy;
 
 
-	public CharacterSheet CurrentCharacterSettings;
+	[SyncVar] public CharacterSheet CurrentCharacterSettings;
```

Comment:
does it really grab the he and she stuff from the client?

Reply:
according to Atner's tests, yes it does.